crates/formatter/tests/cases/array_alignment/after.php

```diff
@@ -0,0 +1,219 @@
+<?php
+
+show_table([ // This is a comment
+    ['Month',     'Premium', 'Revenue'], // This is a comment
+    ['January',   '$0.00',   '$0.00'],
+    ['February',  '$0.00',   '$0.00'],
+    ['March',     '$0.00',   '$0.00'],
+    ['April',     '$0.00',   '$0.00'],
+    ['May',       '$0.00',   '$0.00'],
+    ['June',      '$0.00',   '$0.00'],
+    ['July',      '$0.00',   '$0.00'],
+    ['August',    '$0.00',   '$0.00'],
+    // September ..
+    ['September', '$0.00',   '$0.00'],
+    /// Weeeee!
+    /// Weee!!!!
+    /* Weeeee! */
+    ['October',   '$0.00',   '$0.00'],
+    ['November',  '$0.00',   '$0.00'],
+    ['December',  '$0.00',   '$0.00'], // This is a comment
+]); // This is a comment
+
+show_table([ // This is a comment
+    ['Hello', 11212, 112.1, true,  $bar,  PHP_VERSION],
+    ['World', 125.1, 12,    false, $quux, PHP_VERSION_ID],
+]);
+
+show_table([[[ // This is a comment
+    ['Hello', 11212, 112.1, true,  $bar,  PHP_VERSION],
+    ['World', 125.1, 12,    false, $quux, PHP_VERSION_ID],
+]]]);
+
+show_table(array( // This is a comment
+    array('Month',     'Premium', 'Revenue'), // This is a comment
+    array('January',   '$0.00',   '$0.00'),
+    array('February',  '$0.00',   '$0.00'),
+    array('March',     '$0.00',   '$0.00'),
+    array('April',     '$0.00',   '$0.00'),
+    array('May',       '$0.00',   '$0.00'),
+    array('June',      '$0.00',   '$0.00'),
+    array('July',      '$0.00',   '$0.00'),
+    array('August',    '$0.00',   '$0.00'),
+    array('September', '$0.00',   '$0.00'),
+    array('October',   '$0.00',   '$0.00'),
+    array('November',  '$0.00',   '$0.00'),
+    array('December',  '$0.00',   '$0.00'), // This is a comment
+)); // This is a comment
+
+show_table([ // This is a comment
+    array('Hello', 11212, 112.1,  true,  $bar,  PHP_VERSION),
+    array('World', 125.1, 12,     false, $quux, PHP_VERSION_ID),
+    array('!!',    125.1, 123512, false, $bar,  PHP_VERSION_ID),
+]);
+
+// This table contains a very long string so it won't be formatted as a table
+show_table([ // This is a comment
+    array(
+        'HelloHelloHelloHelloHelloHelloHelloHelloHelloHelloHelloHelloHelloHelloHelloHello',
+        11212,
+        112.1,
+        true,
+        $bar,
+        PHP_VERSION,
+    ),
+    array('World', 125.1, 12, false, $quux, PHP_VERSION_ID),
+    array('!!', 125.1, 123512, false, $bar, PHP_VERSION_ID),
+]);
+
+// too small for table.
+$a = [[1, 2], [3, 4], [5, 6]];
+
+$arr = [
```

Comment:
is this RTL text?
it looks "not aligned" anyway?

Reply:
Yes, GitHub does not render it well 😓